src/@types/aquarius/SearchQuery.ts

```diff
@@ -40,6 +40,14 @@ declare global {
     }
   }
 
+  interface FilterRange {
+    range: {
+      [property: string]: {
+        [property: string]: string | number
```

Comment:
when i think about a range, i'm thinking mostly in numbers, how can this be a string?

Reply:
This is how Elasticsearch works, there are queries that accept `true` or `"true"`

Reply:
Made it a number. Fixed!